js/app.js

```diff
@@ -1,15 +1,11 @@
-import { firstOrDefaultValue, firstOrDefaultContent } from "./parsing.js";
-import { buildBlogItemsUrl, buildTaxonomiesUrl, fetchKontent } from "./kontent.js";
+import { loadBlogItems } from "./blog-listing.js";
+import { loadAsideBlogTags } from "./blog-tags.js";
 
-const baseUrl = new URL(location);
+export const baseUrl = new URL(location);
```

Comment:
do you need to export stuff from app.js? What is consuming this API?

Reply:
Yes, of course you are right. I have moved these accordingly.